bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -205,59 +205,73 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       msg = s"""Ephemeris for target "${t.name}" is undefined."""
     } yield new Problem(Severity.Warning, msg, "Targets", s.inTargetsView(_.edit(t)))
 
-    def bpIsLgs(b: BlueprintBase): Boolean = {
-      val lgs = b match {
-        case a: GmosNBlueprintBase         => a.altair.ao.some
-        case a: GnirsBlueprintImaging      => a.altair.ao.some
-        case a: GnirsBlueprintSpectroscopy => a.altair.ao.some
-        case a: NifsBlueprintAo            => a.altair.ao.some
-        case a: NiriBlueprint              => a.altair.ao.some
-        case a: GsaoiBlueprint             => a.ao.some
-        case _                             => None
-      }
-      lgs.collect {
-        case AoLgs => true
-        case _     => false
-      }.getOrElse(false)
+    def aoPerspectiveIsLgs(ao: AoPerspective): Boolean = ao match {
+      case AoLgs => true
+      case _     => false
     }
 
-    private val lgsCC50Check = for {
+    def bpIsLgs(b: BlueprintBase): Boolean =
+      aoPerspectiveIsLgs(b match {
+        case a: GmosNBlueprintBase         => a.altair.ao
+        case a: GnirsBlueprintImaging      => a.altair.ao
+        case a: GnirsBlueprintSpectroscopy => a.altair.ao
+        case a: NifsBlueprintAo            => a.altair.ao
+        case a: NiriBlueprint              => a.altair.ao
+        case a: GsaoiBlueprint             => a.ao
+        case _                             => AoNone
+      })
+
+    // NOTE: This needs to be maintained for any future instruments that use GeMS.
+    def bpIsGemsLgs(b: BlueprintBase): Boolean =
+      b match {
+        case a: GsaoiBlueprint => aoPerspectiveIsLgs(a.ao)
+        case _                 => false
+      }
+
+    private val lgsIQ70Check = for {
       o  <- p.observations
       c  <- o.condition
       b  <- o.blueprint
-      if bpIsLgs(b) && (c.iq != ImageQuality.IQ70 && c.iq != ImageQuality.BEST)
-    } yield new Problem(Severity.Error, s"LGS requires IQ70 or better", "Observations", s.inObsListView(o.band, _.Fixes.fixConditions(c)))
+      if bpIsLgs(b) && (!bpIsGemsLgs(b)) && (!List(ImageQuality.IQ70, ImageQuality.BEST).contains(c.iq))
```

Comment:
The new `Problem` requires IQ85 or better for GeMS. Seems like it would be erroneous to display this problem since IQ70 is no longer the requirement for GeMS?

That is why I separated the two.